acceptance/suites/pre_suite/upgrade/10_install_puppet.rb

```diff
@@ -1,5 +1,5 @@
-step "Install PC1 repository" do
-  install_puppetlabs_release_repo(hosts, 'pc1')
+step "Install puppet repository" do
+  install_puppetlabs_release_repo(hosts, 'puppet')
```

Comment:
@adrienthebo Correct me if I'm wrong, but I think the way these tests are currently built, we're expecting to do an upgrade from Puppet 4 / Puppet Server 2.x to Puppet / Puppet Server 5 (latest).  If we were to change this line to use the new "puppet" repo for Puppet 5 rather than "pc1", I wonder if these tests would break.  WDYT?